src/include/security/widgets.rb

```diff
@@ -271,12 +273,24 @@ def boot_option_labels
       }
     end
 
+    def shutdown_labels
+      {
```

Comment:
NP: This looks like a constant, but with translations it [is a bit tricky](https://github.com/webyast/webyast/wiki/Localization#translating-constants).